core/src/main/java/io/lionweb/language/Language.java

```diff
@@ -142,29 +142,30 @@ public Concept requireConceptByName(String name) {
         .orElse(null);
   }
 
-  public String getName() {
+  public @Nullable String getName() {
     return this.getPropertyValue("name", String.class);
   }
 
   @Override
-  public String getKey() {
+  public @Nullable String getKey() {
     return this.getPropertyValue("key", String.class);
   }
 
-  @Nullable
-  public String getVersion() {
+  public @Nullable String getVersion() {
```

Comment:
This is a language's **version** which definitely shouldn't be `null`. (It's also not the name.)